src/client.js

```diff
@@ -403,16 +403,24 @@ module.exports = class IrcClient extends EventEmitter {
                 })
             ];
 
-            blocks.forEach(block => this.raw(commandName, target, block));
+            blocks.forEach(block => {
+                if (tags) {
```

Comment:
we should probably check for tags.length

Reply:
Done, tags would be an object so I added Object.keys(tags).length